ray-operator/config/samples/ray-service.mobilenet.yaml

```
@@ -0,0 +1,72 @@
+apiVersion: ray.io/v1alpha1
+kind: RayService
+metadata:
+  name: rayservice-mobilenet
+spec:
+  serviceUnhealthySecondThreshold: 300 # Config for the health check threshold for service. Default value is 60.
+  deploymentUnhealthySecondThreshold: 300 # Config for the health check threshold for deployments. Default value is 60.
+  serveConfig:
+    importPath: mobilenet.graph
```

```suggestion
    importPath: mobilenet:graph
```
Both are equivalent, but this is more consistent with the rest of our examples.

---

This is outdated. I have already updated the `importPath` based on the review comments of https://github.com/ray-project/serve_config_examples/pull/1.